pkg/docker/docker.go

```diff
@@ -18,8 +18,8 @@ const (
 	ScriptsURLEnvironment = "STI_SCRIPTS_URL"
 	LocationEnvironment   = "STI_LOCATION"
 
-	ScriptsURLLabel  = "io.s2i.scripts-url"
```

Comment:
I don't like to leave a deprecated constant here with value that we claim as 'deprecated' here. I think it will be more confusing... I'm OK with leaving a comment on top of this new value saying that this is a new value and the old one is deprecated.